venom/src/test/java/com/github/venom/service/ServiceDelegateTest.kt

```diff
@@ -26,17 +28,20 @@ class ServiceDelegateTest {
     @Test
     fun startService_startServiceIfNotRunning() {
         setRunningState(false)
-        delegate.startService()
-        // it's not possible to verify the Intent unfortunately
-        verify { context.startService(any()) }
+        mockkStatic(androidx.core.content.ContextCompat::class) {
```

Comment:
Let's import `ContextCompat` class to avoid writing the full package name here.

Reply:
Sorry for all these commits, I was doing this directly from GitHub...